<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set_radar_filter</class>
 <widget class="QDialog" name="set_radar_filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set radar filter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="terminal.qrc">
    <normaloff>:/images/set_radar_filter.png</normaloff>:/images/set_radar_filter.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>        Gate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="filter_index">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>31</y>
     <width>149</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filter_Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="filter_type">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cluster filter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Object filter</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Filter_Index</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="filter_active">
      <property name="text">
       <string>Filter_Active</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="filter_valid">
      <property name="text">
       <string>Filter_Valid</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>170</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>gate_min</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>gate_max</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="unit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="clear_all_filters">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Filters</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="terminal.qrc"/>
 </resources>
 <connections/>
</ui>
